modules/images-imagestream-periodic-import-list.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * openshift_images/image-streams-managing.adoc
+
+:_content-type: PROCEDURE
+[id="images-imagestream-periodic-import-list_{context}"]
+= Configuring periodic importing of manifest lists
+
+To periodically re-import a manifest list, you can use the `--scheduled` flag.
+
+.Procedure
+
+* Set the image stream to periodically update the manifest list by entering the following command:
++
+[source,terminal]
+----
+$ oc import-image <multiarch-image-stream-tag>  --from=<registry>/<username>/<image-name> \
```

Comment:
Better to keep consistent, use <project_name> in all the docs.
And https://docs.openshift.com/container-platform/4.12/openshift_images/image-streams-manage.html#importing-manifest-list-through-imagestreamimport_image-streams-managing need update       ```name: <registry>/<user_name>/<image_name>``` too

Reply:
@xiuwang fixed across the docs.